src/clients/api/queries/getHypotheticalAccountLiquidity.ts

```diff
@@ -0,0 +1,34 @@
+import BigNumber from 'bignumber.js';
+import { Contract } from 'web3-eth-contract';
+import { Asset } from 'types';
+
+export interface IGetHypotheticalAccountLiquidityInput {
+  comptrollerContract: Contract; // @TODO: use contract type (through Typechain?)
+  account: string | null | undefined;
+  asset: Asset;
+  balanceOf: string;
+  borrowAmount?: number | BigNumber;
+}
+
+export type IGetHypotheticalAccountLiquidityOutput = number;
```

Comment:
`GetHypotheticalAccountLiquidityInput` and `GetHypotheticalAccountLiquidity` (to make them as types, not interfaces).

Reply:
Interface is the correct option here. We are describing the shape of data/ object. Types are good for aliasing types of data (unions, primatives, other types ect.)

Reply:
Right, but the second type isn't an interface. You posted this exact comment on my PR actually ahah

Reply:
🤦 I read that wrong

Reply:
Actually I think it was meant as a response here - https://github.com/VenusProtocol/venus-protocol-interface/pull/264#discussion_r845817589